drv/gimlet-seq-server/src/main.rs

```diff
@@ -90,7 +90,13 @@ enum Trace {
     RailsOn,
     UartEnabled,
     A0(u16),
-    SetState(PowerState, PowerState, u64),
+    SetState {
```

Comment:
**praise:** Seems like a really good change to add the field names here.

Reply:
Yeah, tuple-like enum variants with two fields of the same type are almost always a bad move IMHO...